src/backend/marsha/core/api/live_registration.py

```diff
@@ -118,6 +127,40 @@ def get_throttles(self):
 
         return [throttle() for throttle in throttle_class]
 
+    def create(self, request, *args, **kwargs):
```

Comment:
You should override `perform_create` instead of create IMO.

```
def perform_create(self, serializer):
   super().perform_create(serializer)
   # send the mail, catch error and update the model if needed
```

Reply:
true it's much better